app/src/main/java/com/github/dfa/diaspora_android/web/CustomWebViewClient.java

```diff
@@ -56,6 +57,11 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
                 || (host != null && (url.startsWith("https://" + host)
                 || url.startsWith("http://" + host)))) {
             return false;
+        }//make youtube links open external-->never customtab
+        if (url.startsWith("https://youtube")||url.startsWith("https://www.youtube")) {
```

Comment:
* make this `else if`, other wise the `else` is not anymore functioning like it should
* Please check here also for `m.youtube.com`